sdk/recoveryservices/Microsoft.Azure.Management.RecoveryServices/src/Generated/Models/VaultProperties.cs

```diff
@@ -50,11 +50,16 @@ public VaultProperties()
         /// 'PartialSuccess'</param>
         /// <param name="backupStorageVersion">Backup storage version. Possible
         /// values include: 'V1', 'V2', 'Unassigned'</param>
+        /// <param name="publicNetworkAccess">property to enable or disable
+        /// resource provider inbound network traffic from public clients.
+        /// Possible values include: 'Enabled', 'Disabled'</param>
         /// <param name="monitoringSettings">Monitoring Settings of the
         /// vault</param>
         /// <param name="redundancySettings">The redundancy Settings of a
         /// Vault</param>
-        public VaultProperties(string provisioningState = default(string), UpgradeDetails upgradeDetails = default(UpgradeDetails), IList<PrivateEndpointConnectionVaultProperties> privateEndpointConnections = default(IList<PrivateEndpointConnectionVaultProperties>), string privateEndpointStateForBackup = default(string), string privateEndpointStateForSiteRecovery = default(string), VaultPropertiesEncryption encryption = default(VaultPropertiesEncryption), VaultPropertiesMoveDetails moveDetails = default(VaultPropertiesMoveDetails), string moveState = default(string), string backupStorageVersion = default(string), MonitoringSettings monitoringSettings = default(MonitoringSettings), VaultPropertiesRedundancySettings redundancySettings = default(VaultPropertiesRedundancySettings))
+        /// <param name="securitySettings">Security Settings of the
+        /// vault</param>
+        public VaultProperties(string provisioningState = default(string), UpgradeDetails upgradeDetails = default(UpgradeDetails), IList<PrivateEndpointConnectionVaultProperties> privateEndpointConnections = default(IList<PrivateEndpointConnectionVaultProperties>), string privateEndpointStateForBackup = default(string), string privateEndpointStateForSiteRecovery = default(string), VaultPropertiesEncryption encryption = default(VaultPropertiesEncryption), VaultPropertiesMoveDetails moveDetails = default(VaultPropertiesMoveDetails), string moveState = default(string), string backupStorageVersion = default(string), string publicNetworkAccess = default(string), MonitoringSettings monitoringSettings = default(MonitoringSettings), VaultPropertiesRedundancySettings redundancySettings = default(VaultPropertiesRedundancySettings), SecuritySettings securitySettings = default(SecuritySettings))
```

Comment:
Breaking change. `publicNetworkAccess` is inserted in existing parameter list, which could cause breaking change for existing customer codes.